pkgs/applications/audio/pragha/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, dbus-glib
+, desktop-file-utils
+, fetchFromGitHub
+, glyr
+, gst_all_1
+, gtk3
+, hicolor-icon-theme
+, intltool
+, libcddb
+, libcdio
+, libcdio-paranoia
+, libgudev
+, liblastfmSF
+, libnotify
+, libpeas
+, makeWrapper
+, mkDerivation
+, pcre
+, pkg-config
+, qtbase
+, sqlite
+, taglib
+, which
+, xfce
+, zlib
+}:
+
+mkDerivation rec {
+  pname = "pragha";
+  version = "1.3.4";
+
+  src = fetchFromGitHub {
+    owner = "pragha-music-player";
+    repo = "pragha";
+    rev = "v${version}";
+    sha256 = "sha256:0n8gx8amg5l9g4w7s4agjf8mlmpgjydgzx3vryp9lzzs9xrd5vqh";
+  };
+
+  CPPFLAGS = [ "-DHAVE_PARANOIA_NEW_INCLUDES" ];
+
+  nativeBuildInputs = [
+    intltool
+    pkg-config
+    xfce.xfce4-dev-tools
+    makeWrapper
+  ];
+
+  buildInputs = [
+    dbus-glib
+    desktop-file-utils
+    glyr
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gstreamer
+    gtk3
+    hicolor-icon-theme
+    libcddb
+    libcdio
+    libcdio-paranoia
+    libgudev
+    liblastfmSF
+    libnotify
+    libpeas
+    pcre
+    qtbase
+    sqlite
+    taglib
+    zlib
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/pragha \
+      --prefix PATH ":" ${lib.makeBinPath [ "$out" which  glyr ]} \
```

Comment:
Would it be a good idea to move the packages from $out to $out/bin which is in PATH by default?